client/web/src/enterprise/batches/detail/BatchChangeDetailsTabs.tsx

```diff
@@ -131,7 +137,14 @@ export const BatchChangeDetailsTabs: React.FunctionComponent<BatchChangeDetailsT
                             <Icon className="text-muted mr-1" as={FileDocumentIcon} />{' '}
                             <span className="text-content" data-tab-content="Executions">
                                 Executions
-                            </span>
+                            </span>{' '}
+                            <Badge
+                                variant={executingCount === 0 ? 'secondary' : 'warning'}
+                                pill={true}
+                                className="ml-1"
+                            >
+                                {executingCount} {batchChange.batchSpecs.pageInfo.hasNextPage && <> + </>}
```

Comment:
I don't think there should be a space before the number, wdyt? `20+` vs `20 +`
```suggestion
                                {executingCount}{batchChange.batchSpecs.pageInfo.hasNextPage && <>+</>}
```

Reply:
no space! removed it, thanks